### ISSUE TITLE
Add M4A Support

### ISSUE THREAD

**Issue by zerobun** [Sandakan/Nora#247]
Please add M4A format support, 70% of my music is in this format, and I'm sure a lot out there use M4A format too.
Thanks in advance 🙂

**Comment by Invader202** [comment]
i just opened github to open this issue ☠

**Comment by Sandakan** [comment]
@Invader202 @zerobun ,
M4A support is possible for the next update, but I will need some time to check everything out.

**Comment by IkelAtomig** [comment]
> Please add M4A format support, 70% of my music is in this format, and I'm sure a lot out there use M4A format too. Thanks in advance 🙂

Yeah, it is a lot of headache to convert them to mp3 and append metadata again.
